### PR TITLE
add xiaomi 4c and z11support

### PR DESCRIPTION
**Pull request by ililaoban**

### CODE REVIEW

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

File: data/devices/xiaomi.yml [hunk 1]
```diff
@@ -42,6 +42,44 @@
     default_brightness: 162
     theme: portrait_hdpi
 
+- name: Xiaomi Mi4C
+  id: libra
+  codenames:
+    - libra
+  architecture: arme64-v8a
```
Comment (chenxiaolong) [inline data/devices/xiaomi.yml]: The build failed because there's a typo here. It should be `arm64-v8a`, not `arme64-v8a`.

**Comment by ililaoban** [comment]
@chenxiaolong ok,i've correct it.

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Comment by chenxiaolong** [comment]
Merged, thanks!

**Pull request merged**